docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+  sirbot:
+    image: pyslackers/sirbot:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    env_file:
+      - .env
+    restart: unless-stopped
+    expose:
+      - "8080"
```

Comment:
Oh I see, I have the site deployed direct at the moment. But sharing an nginx will work - we may just need a compose for everything together if we want to do that and a different config format